modules/fedimint-wallet-client/src/lib.rs

```diff
@@ -627,6 +628,10 @@ impl WalletClientModule {
         ))
     }
 
+    pub async fn btc_tx_has_no_size_limit(&self) -> FederationResult<bool> {
+        Ok(self.module_api.module_consensus_version().await? >= ModuleConsensusVersion::new(2, 2))
+    }
```

Comment:
Later optimization: cache result and use the cached version everywhere. That way this fn call becomes cheap enough to run whenever a user wants to receive money and integrators have to decide if they want to show an on-chain address or not.